frontend/tests/features/collection/collection.test.ts

```diff
@@ -94,7 +94,9 @@ describe("Collection", () => {
     });
   });
 
-  describeIfDevStaging("Deployed Env Tests", () => {
+  describe("Deployed Env Tests", () => {
+    skip(!isDevStagingProd, "BE DOI endpoints only work in dev, staging, prod");
```

Comment:
replace `describeIfDevStaging`, but does the same thing. The bonus is that now we can document why the tests are skipped, which also shows up in the test report

Reply:
what was the initial reason for us avoiding running these tests on prod?

Reply:
Hmm good question! Since the tests don't publish collections, I'm not sure why they can't run in prod 🤔 

@frano-m do you remember why these tests don't run in prod? Thank you!

Reply:
hmm that's a good point! Maybe so we won't pollute the prod database in any case 🤔 But even if it does, it'd be pretty easy to clean up all private collections associated with the test account once in a while too, and it shouldn't slow down other users' experience too, since it's all private collections?

I'll merge the PR for now 👌 

Thanks all!